components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/resources/idp.yaml

```diff
@@ -1909,6 +1909,9 @@ components:
         alias:
           type: string
           example: 'https://localhost:9444/oauth2/token'
+        idpIssuerName:
```

Comment:
since we are changing the request/response formats, we might have to release a new API version in public branch.
@IsuraD @ayshsandu WDYT?

Reply:
is this an optional attribute? Use in GET or POST?

Reply:
Yes, this is an optional attribute in POST request